balarco/utils.py

```diff
@@ -0,0 +1,83 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import viewsets, status
+from rest_framework.response import Response
+from rest_framework.authentication import TokenAuthentication, SessionAuthentication
+
+
+def generic_rest_list_objects(request, serializer_class, obj_class):
+    queryset = obj_class.objects.filter(is_active=True)
+    serializer = serializer_class(queryset, many=True)
+    return Response(serializer.data, status.HTTP_200_OK)
+
+
+def generic_rest_create_object(request, serializer_class, obj_class):
+    serializer = serializer_class(data=request.data)
+    if serializer.is_valid():
+        new_obj = obj_class.objects.create(**serializer.validated_data)
+        return Response(serializer_class(new_obj).data, status=status.HTTP_201_CREATED)
+    return Response({
+        'status': 'Bad request',
+        'message': '%s could not be created with received data.' % obj_class.__name__
```

Comment:
For all the message methods here you should use [format](https://www.digitalocean.com/community/tutorials/how-to-use-string-formatters-in-python-3) function, to keep consistency with the project and keep it more [pythonic](https://stackoverflow.com/questions/5082452/python-string-formatting-vs-format).